pkg/crc/machine/machine.go

```diff
@@ -426,6 +425,24 @@ func Start(startConfig StartConfig) (StartResult, error) {
 
 	time.Sleep(time.Minute * 3)
 
+	logging.Info("Updating kubeconfig")
+
+	if err := errors.RetryAfter(60, func() error {
+		status, err := cluster.GetClusterOperatorStatus(ocConfig, "authentication")
+		if err != nil {
+			return &errors.RetriableError{Err: err}
+		}
+		if isReady(status) {
+			return nil
+		}
+		return &errors.RetriableError{Err: errors.New("cluster operator authentication not ready")}
+	}, 2*time.Second); err != nil {
```

Comment:
@gbraad This extra 2 mins in the best scenario doesn't going to increase the startup time but in worst scenario it will increase the startup time by 2 mins.

Reply:
OK

The time of 10 minutes is the optimal path, but thanks for informing.